apps/tenant-management-webapp/src/app/pages/admin/services/script/editor/tombstone.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { FunctionComponent, useEffect, useRef, useState } from 'react';
+
+import { TombStoneWrapper, Edit, Tooltip } from '../styled-components';
+import { ScriptItem } from '@store/script/models';
+import { GoAIconButton } from '@abgov/react-components-new';
+import { AddScriptModal } from '../addScriptModal';
+
+import { useSelector } from 'react-redux';
+import { tenantRolesAndClients } from '@store/sharedSelectors/roles';
+
+interface props {
+  selectedScript: ScriptItem;
+  onSave;
+}
+export const TombStone: FunctionComponent<props> = ({ selectedScript, onSave }) => {
```

Comment:
A very small issue here. I have not see children property here. So, I would suggest replacing the FunctionComponent with normal Element.JSX.